themes/default/layouts/registry/package.html

```diff
@@ -0,0 +1,123 @@
+{{ define "main" }}
```

Comment:
thank you so much for consolidating this to one file now. this was much needed and i think will make this so much easier to work with going forward :star_struck: